README.md

```diff
@@ -6,190 +6,225 @@
-  * [Add a new rewrite rule](include/onnxruntime/core/optimizer/rewrite_rule.h)
-  * [Add an execution provider](docs/AddingExecutionProvider.md)
+ONNX Runtime is up to date and backwards compatible with all operators (both DNN and traditional ML) since ONNX v1.2.1+. [(ONNX compatibility details)](docs/Versioning.md). Newer versions of ONNX Runtime support all models that worked with prior versions, so updates should not break integrations. 
 
-***
+* [Supported operators/types](./docs/OperatorKernels.md)
+  * *Operators not supported in the current ONNX spec may be available as a [Contrib Operator](./docs/ContribOperators.md)*
+* [Extensibility: Add a custom operator/kernel](docs/AddingCustomOp.md)
 
-# Installation
-**Quick Start:** The [ONNX-Ecosystem Docker container image](https://github.com/onnx/onnx-docker/tree/master/onnx-ecosystem) is available on Dockerhub and includes ONNX Runtime (CPU, Python), dependencies, tools to convert from various frameworks, and Jupyter notebooks to help get started. Additional dockerfiles can be found [here](./dockerfiles).
+### Binaries
 
-## API Documentation
+Official builds are available on PyPi (Python) and Nuget (C#/C/C++):
 
-|Language|Supported Versions|Samples|
-|--|--|--|
+* Default CPU Provider (Eigen + MLAS)
+* GPU Provider - NVIDIA CUDA
+* GPU Provider - DirectML (Windows)
+  * *On Windows, the [DirectML execution provider](./docs/execution_providers/DirectML-ExecutionProvider.md) is recommended for optimal performance and compatibility with a broad set of GPUs.*
+
+Dev builds created from the master branch are available for testing newer changes between official releases. Please use these at your own risk. We strongly advise against deploying these to production workloads as support is limited for dev builds.
+
+|Pypi (Python)|Nuget (C#/C/C++)|Other package repositories|
+|---|---|---|
+*If using pip, run `pip install --upgrade pip` prior to downloading.*<br><br>CPU: [**onnxruntime**](https://pypi.org/project/onnxruntime) / [ort-nightly (dev)](https://test.pypi.org/project/ort-nightly)<br><br>GPU: [**onnxruntime-gpu**](https://pypi.org/project/onnxruntime-gpu) / [ort-gpu-nightly (dev)](https://test.pypi.org/project/ort-gpu-nightly) | CPU: [**Microsoft.ML.OnnxRuntime**](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime) / [ort-nightly (dev)](https://aiinfra.visualstudio.com/PublicPackages/_packaging?_a=feed&feed=ORT-Nightly) <br><br>GPU: [**Microsoft.ML.OnnxRuntime.Gpu**](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.gpu) / [ort-nightly (dev)](https://aiinfra.visualstudio.com/PublicPackages/_packaging?_a=feed&feed=ORT-Nightly)</li></ul>|[Contributed non-official packages](https://docs.microsoft.com/en-us/windows/ai/windows-ml/get-started-uwp) (including Homebrew, Linuxbrew, and nixpkgs)<br><br>*These are not maintained by the core ONNX Runtime team and may have limited support; use at your discretion.*|
+
+#### System Requirements
+
+* Visual Studio
```

Comment:
Is Visual Studio actually a system requirement? This might be confusing for users who don't use windows